src/presets/vercel.ts

```diff
@@ -11,38 +11,23 @@ export const vercel = defineNitroPreset({
   entry: '#internal/nitro/entries/vercel',
   output: {
     dir: '{{ rootDir }}/.vercel/output',
-    serverDir: '{{ output.dir }}/functions/index.func',
+    serverDir: '{{ output.dir }}/functions/__nuxt.func',
```

Comment:
Would using index as function name make conflicts?

Reply:
(it could be __nitro or __server in this case)

Reply:
Yes, it does; that's why I changed it. 👍

Reply:
oh of course :facepalm: